src/components/BigFlashCard.js

```diff
@@ -1,5 +1,10 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 
 const BigFlashCard = (props) => <div className='BigFlashCard'>{props.children}</div>
 
+BigFlashCard.PropTypes = {
+  children: PropTypes.object
```

Comment:
`children` are a special prop -- they are the components contained inside this component. We leave them out of PropTypes checking.